pkg/yurtappmanager/local/manager.yaml

```diff
@@ -803,3 +934,22 @@ webhooks:
     - UPDATE
     resources:
     - uniteddeployments
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: yurtapp-webhook-service
+      namespace: kube-system
+      path: /validate-apps-openyurt-io-v1alpha1-yurtingress
```

Comment:
lack of "mutate-apps-openyurt-io-v1alpha1-yurtingress" ?

Reply:
only validating webhook is implemented, no mutating webhook.

Reply:
`mutate-apps-openyurt-io-v1alpha1-yurtingress` shoud auto generated by  pkg/yurtappmanager/webhook/yurtingress/mutating/webhooks.go: 
`/ +kubebuilder:webhook:path=/mutate-apps-openyurt-io-v1alpha1-yurtingress,mutating=true,failurePolicy=fail,groups=apps.openyurt.io,resources=yurtingresses,verbs=create;update,versions=v1alpha1,name=myurtingress.kb.io`

Reply:
Yes, it is auto generated, and the generated codes are kept as well. But if we don't need to modify the request from user, why not leave this mutating webhook disabled? It might reduce the communication with webhook server as well.

Reply:
ok, will add mutating webhook to keep consistent with other components. :-)

Reply:
> Yes, it is auto generated, and the generated codes are kept as well. But if we don't need to modify the request from user, why not leave this mutating webhook disabled? It might reduce the communication with webhook server as well.

yes, if we don't need to modify the request from user , we need let this mutating webhook disabled 。  So we need to comment out this line `/ +kubebuilder:webhook:path=/mutate-apps-openyurt-io-v1alpha1-yurtingress,mutating=true,failurePolicy=fail,groups=apps.openyurt.io,resources=yurtingresses,verbs=create;update,versions=v1alpha1,name=myurtingress.kb.io`   in `pkg/yurtappmanager/webhook/yurtingress/mutating/webhooks.go` file.  Otherwise, the mutate configuration will be generated in manager.yaml each time `make generate-manifests` is executed .